setup.py

```diff
@@ -6,22 +6,23 @@
 # HELPER FUNCTIONS #############################################################
 ################################################################################
 
+
 def get_lookup():
-    '''get version by way of urlchecker.version, returns a 
+    """get version by way of urlchecker.version, returns a
     lookup dictionary with several global variables without
     needing to import singularity
-    '''
+    """
     lookup = dict()
-    version_file = os.path.join('urlchecker', 'version.py')
+    version_file = os.path.join("urlchecker", "version.py")
     with open(version_file) as fd:
         exec(fd.read(), lookup)
     return lookup
 
 
 # Read in requirements
-def get_reqs(lookup=None, key='INSTALL_REQUIRES'):
-    '''get requirements, mean reading in requirements and versions from
-    the lookup obtained with get_lookup'''
+def get_reqs(lookup=None, key="INSTALL_REQUIRES"):
```

Comment:
@SuperKogito I think you have the codecov account? It should not be linting the setup.py for issues, we typically don't test that :)

Reply:
okay, I've removed checking the setup.py, so we should be good! It looks like coverage went down a bit for one or two of the files - @SuperKogito let me know what tests you'd like to add (or if we should safely ignore).